mirth-test.sh

```diff
@@ -28,12 +28,12 @@ FAILED=0
 
 set +e
 
-for filename in $(find src/tests -name "*.mth" -type f -printf '%f\n')
+for filename in $(ls src/mirth-tests | grep .mth)
```

Comment:
Back to using `ls` here because `find` doesn't work with my (janky) workflow on windows.
Also it seems like this was skipping the `data-syntax.mth` test for some reason, unsure.